src/anyon/anyon.jl

```diff
@@ -200,6 +203,41 @@ set_of_native_gates = [
     ControlZ,
 ]
 
+function get_metadata(client::Client, qpu::UnionAnyonQPU)::Metadata
+
+    # TODO: the metadata will eventually be fetched from the host server.
+    # It is hardcoded for the time being
+
+    generation = "Yukon"
+    serial_number = "ANYK202201"
+    excluded_positions = Vector{Int}()
+
+    if qpu isa AnyonYamaskaQPU
+        generation = "Yamaska"
+        serial_number = "ANYK202301"
+    end
+
+    output = Metadata(
+        "manufacturer" => "Anyon Systems Inc.",
+        "generation" => generation,
+        "serial_number" => serial_number,
+        "project_id" => get_project_id(qpu),
+        "qubit_count" => get_num_qubits(qpu.connectivity),
+        "connectivity_type" => get_connectivity_label(qpu.connectivity),
+    )
+
+    realm = get_realm(qpu)
+    if realm != ""
+        output["realm"] = realm
+    end
+
+    if !isempty(excluded_positions)
+        output["excluded_positions"] = excluded_positions
```

Comment:
We need a test for this.

Reply:
this path is unreachable unless we inject excluded_positions, which seems out of place. I'll take it out for now, we can change it back when we implement the API call